client/my-sites/plans/ecommerce-trial/upgrade-confirmation/index.tsx

```diff
@@ -1,32 +1,39 @@
 import { useTranslate } from 'i18n-calypso';
 import Main from 'calypso/components/main';
-import './style.scss';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
 import ConfirmationTask from './confirmation-task';
 import { getConfirmationTasks } from './confirmation-tasks';
 
+import './style.scss';
+
 const TrialUpgradeConfirmation = () => {
 	const translate = useTranslate();
 
 	const tasks = getConfirmationTasks( { translate } );
 
 	return (
-		<Main wideLayout>
-			<div className="trial-upgrade-confirmation__header">
-				<h1 className="trial-upgrade-confirmation__title">
-					{ translate( 'Woo! Welcome to eCommerce' ) }
-				</h1>
-				<div className="trial-upgrade-confirmation__subtitle">
-					{ translate( 'Your purchase has been completed and you’re on the Commerce plan.' ) }
-					<br />
-					{ translate( "Now it's time to get creative. What would you like to do next?" ) }
+		<>
+			<BodySectionCssClass bodyClass={ [ 'trial-upgraded' ] } />
```

Comment:
Why not `trial-upgrade-confirmation` to match the class? We could also use `.ecommerce-trial-upgraded` to be a bit less likely to collide with another usage/context.
```suggestion
			<BodySectionCssClass bodyClass={ [ 'trial-upgrade-confirmation' ] } />
```